pyfiglet/__init__.py

```diff
@@ -785,12 +808,22 @@ def main():
                       help='show installed fonts list')
     parser.add_option('-i', '--info_font', action='store_true', default=False,
                       help='show font\'s information, use with -f FONT')
+    parser.add_option('-c', '--foreground_color', default='DEFAULT', choices=list(COLORS.keys()),
+                      help='prints text with passed foreground color')
+    parser.add_option('-b', '--background_color', default='DEFAULT', choices=list(COLORS.keys()),
+                      help='prints text with passed background color')
+    parser.add_option('-L', '--list_color', action='store_true', default=False,
+                      help='show colors list')
```

Comment:
If possible, it's good to minimise the number of options. No user wants to skim a long list of options, and I'm not sure having an additional three options here is necessary. Bear in mind that we already have 8. How about having a single `--color` option, and perhaps making it understand `--color=list`, `--color=foreground` or `--color=foreground:background`?

Reply:
Done!